tests/Unit/DataStructures/Test_Variables.cpp

```diff
@@ -220,7 +170,7 @@ void test_variables_construction_and_access() noexcept {
 
   // Test stream operator
   const std::string expected_output =
-      "tensor name:\n"
+      "tensor:\n"
```

Comment:
What's going on here?  This commit changes the expected output in the test without changing the code producing that output.

Reply:
I meant to remove the `name()` functions in this commit, sorry. Did this in the update